onyo/cli/get.py

```diff
@@ -160,11 +160,11 @@ def get(args: argparse.Namespace) -> None:
     In addition to keys in asset contents, **PSEUDO-KEYS** can be queried and
     matched.
 
-      * ``is_asset_directory``: is the asset an Asset Directory
-      * ``directory``: parent directory of the asset relative to repo root
-      * ``path``: path of the asset relative to repo root
+      * ``onyo.is.directory``: is the asset an Asset Directory
+      * ``onyo.path.parent`` (default alias: ``directory``): parent directory of the asset relative to repo root
+      * ``onyo.path.relative`` (default alias: ``path``): path of the asset relative to repo root
```

Comment:
Yes, because this is just what it was before. The additional query abilities that this effort should ultimately lead to aren't tested yet. Also: Not sure that `file` works properly yet (probably not). 
In any case: I'd like to change/test the querying properly when the approach is applied to directories and templates that are now able to have pseudo-keys as well.

So, didn't want to put it into documentation before that's actually done and tested, even though at least `absolute` should already work.

Reply:
That makes sense. Thanks for explaining. :-)